app/src/main/java/gov/hhs/cdc/trustedintermediary/external/reportstream/ReportStreamLabOrderSender.java

```diff
@@ -16,7 +28,66 @@ private ReportStreamLabOrderSender() {}
 
     @Override
     public void sendOrder(final LabOrder<?> order) {
-        // String json = FHIR.stringify(order.getUnderlyingOrder())
-        // reportStream.sendRequestBody(json)
+        String json = fhir.encodeResourceToJson(order.getUnderlyingOrder());
+        String bearerToken = requestToken();
+        sendRequestBody(json, bearerToken);
+    }
+
+    public void sendRequestBody(@NotNull String json, @NotNull String bearerToken) { // url param?
+        String res;
+        Map<String, String> headers =
+                Map.of(
+                        "Authorization", "Bearer" + bearerToken,
```

Comment:
There should be a space after `Bearer` and before the API key.

```suggestion
                        "Authorization", "Bearer " + bearerToken,
```

Reply:
This is a good find, thanks!